backend/site-api/src/app/site/create_site.rs

```diff
@@ -54,10 +53,6 @@ pub async fn create_site_helper(
         .cache
         .create_or_update_usage(&site_id, &site, site_type)
         .await;
-    context
```

Comment:
Because the metadata cache helper takes care of updating the cache, we don't have to worry about it here.